### ISSUE TITLE
Is this package anything? 

### ISSUE THREAD

**Issue by kevando** [phuochau/react-native-ar#1]
I like rn and want to do something in AR, but this appears to be just a default project, no?

**Comment by phuochau** [comment]
@kevando I'm developing, it will be coming soon.

**Comment by kevando** [comment]
May I ask what you are developing? Will it involve the camera and recognizing the physical space?

**Comment by doguhanokumus** [comment]
Do you need help to make this faster? I am very interested in AR and have experience about React.
I saw your project structure and wanted to recommend checking out latest react-community/create-react-native-app?

**Comment by kevando** [comment]
Yes I agree with @doguhanokumus I am very interested in contributing to this project also.
